common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala

```diff
@@ -17,15 +17,18 @@ object Messages {
 
   sealed trait Message
 
-  final case class DeviceSeen(deviceId: Device.Id,
+  val PartitionPrefix = 1
+
+  final case class DeviceSeen(uuid: Id,
```

Comment:
Actually, I guess the UI can just fetch this data using another request if it's necessary. No need to add info we don't need.